dags/socorro_import.py

```diff
@@ -1,14 +1,35 @@
 from airflow import DAG
 from airflow.operators.subdag_operator import SubDagOperator
+from airflow.contrib.hooks.gcp_api_base_hook import GoogleCloudBaseHook
+from airflow.contrib.operators.bigquery_table_delete_operator import BigQueryTableDeleteOperator # noqa
+from airflow.contrib.operators.s3_to_gcs_transfer_operator import S3ToGoogleCloudStorageTransferOperator # noqa
+
 from datetime import datetime, timedelta
-from operators.emr_spark_operator import EMRSparkOperator
+
+from operators.gcp_container_operator import GKEPodOperator
 from utils.gcp import load_to_bigquery
 from utils.status import register_status
+from utils.dataproc import moz_dataproc_pyspark_runner
+
+"""
+Originally, this job read json(non ndjson) from aws prod at:
+s3://crashstats-telemetry-crashes-prod-us-west-2/v1/crash_report 
+and wrote the data to parquet format in aws dev at:
+s3://telemetry-parquet/socorro_crash/v2
+
+Until we migrate socorro to gcp (https://bugzilla.mozilla.org/show_bug.cgi?id=1512641),
+or at least modify it to write crash stats to gcs, this job will now copy json from s3
+to gcs, use dataproc to rewrite the data to parquet in gcs, and load the parquet data
+into bigquery.
+
+Copying the parquet data back from gcs to s3 will be necessary until we have migrated
+all workloads using the old athena datasource to it's bigquery counterpart.
+"""
```

Comment:
Could you document the necessary variables and connections in the docstring?

Reply:
yes